cli/tests/cli-reference@.md.snap

```diff
@@ -111,6 +112,7 @@ To get started, see the tutorial at https://github.com/martinvonz/jj/blob/main/d
 ###### **Subcommands:**
 
 * `abandon` — Abandon a revision
+* `annotate` — Annotate a file
```

Comment:
Can you describe it without using the command name?

Reply:
I followed the structure of abandon but I can modify it

Reply:
For name ideas, `credit` is somewhat popular in #2988. I also like `attribute` a bit more than `annotate` if we want a neutral tone. Perhaps I'll mention that there.

If people are not super-excited about these, `annotate` is a fine default choice for a name. I hope it will change later, but it doesn't have to.

Reply:
We've got a few options
* annotate: I generally like it
* attribute: also fine with this
* credit: not my favorite, it doesn't really describe to me what it does
* blame: just silly (leftover from git). No reason to be so negative in the command.

Reply:
I just thought of `map`, and then rereading `attribute` I thought of `byline`.

Reply:
map isn't bad actually. I'm starting to like it more and more. 
Especially if we eventually add a TUI to this, it will feel like navigating a file map line by line.

Anyone else want to weigh in? I don't really feel strongly here about the name, but we should decide something before merging

Reply:
I vote for `annotate` because it's familiar.

Reply:
Also prefer `annotate`. It's available in:
- `svn` (also has `blame` and `praise` synonyms),
- `p4` (also has `blame` synonym, I think),
- `hg` (also has `blame` synonym)
- `git` (which uses a different output format from `git blame`, but fundamentally has the same behavior, I believe)

Probably others.

Reply:
I'm happy with annotate